src/main/kotlin/org/rust/ide/annotator/RsExpressionAnnotator.kt

```diff
@@ -59,9 +59,12 @@ class RsExpressionAnnotator : Annotator {
         } else {
             if (missingFields.isNotEmpty()) {
                 val structNameRange = expr.parent.childOfType<RsPath>()?.textRange
-                if (structNameRange != null)
+                if (structNameRange != null) {
                     holder.createErrorAnnotation(structNameRange, "Some fields are missing")
                         .registerFix(AddStructFieldsFix(decl.namedFields, missingFields, expr), expr.parent.textRange)
+                    holder.createErrorAnnotation(structNameRange, "Some fields are missing")
+                        .registerFix(AddStructFieldsFix(decl.namedFields, missingFields, expr, true), expr.parent.textRange)
```

Comment:
For boolean arguments, it's better to specify parameter name at the call site for better readability.